docs/index.html

```diff
@@ -15,7 +15,8 @@
       name: 'Drava',
       repo: 'https://github.com/wangqianwen0418/DRAVA',
       loadSidebar: true,
-      subMaxLevel: 3
+      subMaxLevel: 3,
+      themeColor: '#F7941F'
```

Comment:
Took an orange color from the interface & figures, but feel free to change the color.